notes/SDEverywhere Web App Configuration Reference.md

```diff
@@ -14,36 +14,35 @@ SDEverywhere uses CSV files to specify the configuration of web apps that it gen
 
 The app.csv file contains application global properties.
 
-Column | Type | Description
----|---|---
-title | string | application title
-version | string | version number
-initialView | string | first level view title \> second level view title
-trackSliders | boolean | set true to run the model as sliders are moved; set false to wait until the slider is released to run the model
-initialTime | number | start time in model runs
-startTime | number | start time to show on graphs
-endTime | number | final time in model runs
-externalDatfiles | string | optional semicolon-delimited list of Vensim DAT filenames giving external data variables
-chartDatfiles | string list | optional semicolon-delimited list of Vensim DAT filenames with chart data
-logo | string | optional filename of an app logo image
-helpUrl | string | optional URL of an online help file
+| Column           | Type        | Description                                                                                                     |
```

Comment:
This is kind of an excessive change by Prettier; I like using terse table syntax, but oh well, I guess for cases like this where the text isn't changed often, it's not a big deal.